### PR TITLE
Simplify package

### PR DESCRIPTION
**Pull request by JSchmie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -210,4 +210,8 @@ pyrightconfig.json
 # End of https://www.toptal.com/developers/gitignore/api/python,visualstudiocode,linux
 
 /build
-scraibe_webui.egg-info
\ No newline at end of file
+scraibe_webui.egg-info
+
+costume_config.yaml
```
Comment (mahenning) [inline .gitignore:215]: custom_config.yaml is better I think

File: scraibe_webui/cli.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def cli():
     config = arg_dict.pop("config")
     
     server_kwargs = arg_dict.pop("server_kwargs")
-    
+    # TODO : Change this to the new interface type
```
Comment (mahenning) [inline scraibe_webui/cli.py:44]: Is this todo okay on dev?

File: scraibe_webui/misc/config.yaml [hunk 3]
```diff
@@ -40,20 +40,36 @@ layout:
...
+  smtp_server : null
+  smtp_port : 0
+  sender_password : null
+  context_kwargs : {}
+  default_subject: "SCRAIBE"
+  error_template: scraibe_webui/misc/error_notification_template.html
+  error_subject: An error occured during processing.
+  error_format_options:
+    #exeption is mandatroy for your html  will be set to the related exepttion in the Code
+   contact_email: support@mail.com # set this to your mail if you are using a costume error_template you can add an any amount of format options here.
```
Comment (mahenning) [inline scraibe_webui/misc/config.yaml:62]: "custom"

File: scraibe_webui/misc/config.yaml [hunk 4]
```diff
@@ -40,20 +40,36 @@ layout:
...
+  sender_email : null 
+  smtp_server : null
+  smtp_port : 0
+  sender_password : null
+  context_kwargs : {}
+  default_subject: "SCRAIBE"
+  error_template: scraibe_webui/misc/error_notification_template.html
+  error_subject: An error occured during processing.
+  error_format_options:
+    #exeption is mandatroy for your html  will be set to the related exepttion in the Code
```
Comment (mahenning) [inline scraibe_webui/misc/config.yaml:61]: "exception"

File: scraibe_webui/utils/background.py [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+from scraibe_webui.global_var import MAX_CONCURRENT_MODELS
+import scraibe_webui.global_var as gv
+from .mail import MailService
+from .wrapper import ScraibeWrapper
+
+threadLimiter = BoundedSemaphore(MAX_CONCURRENT_MODELS)
+
+class BoundedThread(Thread):
+    """
+    Cosutom Thread class that limits the number of threads that can run concurrently
```
Comment (mahenning) [inline scraibe_webui/utils/background.py:16]: "Custom" :D

File: scraibe_webui/utils/interactions.py [hunk 6]
```diff
@@ -2,58 +2,240 @@
...
+def get_pipe(keep_model_alive : bool, scraibe_params : dict) -> ScraibeWrapper:
+    """
+    This function loads the model into memory only when it's needed, which is beneficial for occasional use. 
+    By doing so, it efficiently manages resource usage by ensuring that the resources required by the model 
+    are only utilized when the model is actually running. This approach helps to free up resources when the 
+    model is not in use, thereby improving the overall performance and efficiency of the system.
+    
+    Args:
+        timer_interval (int): The interval (in seconds) at which the model should be kept alive in memory.
+        audio (str): The path to the audio file for which the prediction should be made.
```
Comment (mahenning) [inline scraibe_webui/utils/interactions.py:80]: Docstring and params do not match?

**Review by mahenning — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (JSchmie) [reply .gitignore:215]: fixed with next push

File: scraibe_webui/cli.py (hunk 2 above)
Comment (JSchmie) [reply scraibe_webui/cli.py:44]: For now, yes, but we have to change the cli again before release

File: scraibe_webui/utils/mail.py [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+        try:
+            with smtplib.SMTP(self.smtp_server, self.smtp_port) as server:
+                server.starttls(context=self.context)
+                server.login(self.sender_email, self.password)
+            return True
+        except smtplib.SMTPNotSupportedError:
+            Warning("SMTP AUTH extension not supported by server. Try without login.")
+            return False
+    
+    def setup_message(self, subject: str, receiver_email: str, message: str, attachments: list = []) -> MIMEMultipart:
```
[the remark this answers is not in the document]
Comment (JSchmie) [reply scraibe_webui/utils/mail.py:100]: Thanks, fixed using the suggested workflow.

**Review by mahenning — APPROVED** [review]

**Pull request merged**